ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5692,3 +5693,1101 @@ kong_apis:
+  - name: notifications
+    uris: "{{ discussions_prefix }}/notifications"
+    upstream_url: "{{ discussions_mw_service_url }}/notifications"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userDetails
+    uris: "{{ discussions_prefix }}/user/:userslug"
+    upstream_url: "{{ discussions_mw_service_url }}/user/:userslug"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userUpvoteList
+    uris: "{{ discussions_prefix }}/user/:userslug/upvoted"
+    upstream_url: "{{ discussions_mw_service_url }}/user/:userslug/upvoted"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userDownvoteList
+    uris: "{{ discussions_prefix }}/user/:userslug/downvoted"
+    upstream_url: "{{ discussions_mw_service_url }}/user/:userslug/downvoted"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userBookmarksList
+    uris: "{{ discussions_prefix }}/user/:userslug/bookmarks"
+    upstream_url: "{{ discussions_mw_service_url }}/user/:userslug/bookmarks"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: categoryDetails
+    uris: "{{ discussions_prefix }}/category/:category_id/:slug/:topic_index"
```

Comment:
make sure the values provided in this entire PR with :variable are indeed correct!